src/python/build_management/build_manager.py

```diff
@@ -864,6 +864,38 @@ def setup(self):
     return True
 
 
+class CuttlefishKernelBuild(RegularBuild):
+  """Represents a Android Cuttlefish kernel build."""
+
+  def setup(self):
+    """Android kernel build setup."""
+    from platforms.android import adb
+
+    result = super().setup()
+    if not result:
+      return result
+
+    environment.set_value('VMLINUX_PATH', self.build_dir)
+
+    cvd_dir = environment.get_value('CVD_DIR')
+    adb.stop_cuttlefish_device()
+
+    for image_filename in ['bzImage', 'initramfs.img']:
+      # Delete existing kernel image.
+      rm_cmd = f'rm {cvd_dir}/{image_filename}'
+      adb.execute_command(
+          rm_cmd, timeout=adb.RECOVERY_CMD_TIMEOUT, on_cuttlefish_host=True)
```

Comment:
do we need to delete if we're overwriting this anyway on line 892? 

if so, we should also move `image_dest` to before line 885 and use that in the rm command instead.

Reply:
Not necessary, deleted.